cmd/Dockerfile

```diff
@@ -37,7 +37,7 @@ WORKDIR /app
 
 ARG GOARCH=amd64
 # TAGS can be overriden to modify the go build tags (e.g. build without netgo)
-ARG TAGS="netgo"
+ARG TAGS="netgo,osusergo"
```

Comment:
I'm ok with updating the default user, but I think this should go in a different PR since it has implications for node operators and infra